### ISSUE TITLE
Font and @fontface issue

### ISSUE THREAD

**Issue by rcass** [temefleet/Instanews-App#8]
So close to having the font files and @font-face perfect.

So when is comes to the font files you currently are using in your project, you currently have two.  There should be 4 different sets. They were given in the project assets, but just need to be generated for web. 
This is inside of the given assets folder:
![screen shot 2017-08-10 at 2 14 10 pm](https://user-images.githubusercontent.com/5028555/29192417-34cb6ca6-7dd6-11e7-985f-5b0737c848ec.png)

Secondly, the @fontface rules are so close! Just need to change some values to prevent faux bolding. For example are the font-family names should be the same and the font-weight and font-style should change according to that typeface.
[Here](http://css-snippets.com/web-fonts-faux-bold-and-italic/)is an article explaining more about it!

Basically here is what needs to be addressed:

- [ ] Make sure the font folder contains 4 different font sets (they will have multiple files to each for browser support).

- [x] Change @font-face values to prevent faux bolding. 

Just marked this as closed when you are finished, so I can take another look! You are so close, keep going!

**Comment by temefleet** [comment]
Fixed that. Thanks for your help!
